images/nut-upsd/entrypoint.sh

```diff
@@ -44,6 +44,7 @@ EOF
 [$API_USER]
         password = $API_PASSWORD
         upsmon $SERVER
+        instcmds = ALL
```

Comment:
Can we also include set actions?

```suggestion
        actions = SET
        instcmds = ALL
```